android/cli/lib/gradle-wrapper.js

```diff
@@ -379,6 +406,14 @@ async function writeJavaPropertiesFile(filePath, properties) {
 
 	// Create the properties files with the text lines generated above.
 	await fs.writeFile(filePath, fileLines.join('\n') + '\n');
+
+	// Add own configuration is provided. Following a proper format is on own risk!
+	const fileName = filePath.split('/').pop();
+	const gradlePropertiesOverwriteFile = path.join(platformPath, fileName);
+
+	if (fs.existsSync(gradlePropertiesOverwriteFile)) {
+		await fs.appendFile(filePath, fs.readFileSync(gradlePropertiesOverwriteFile, 'utf-8').toString() + '\n');
```

Comment:
Again, for consistency:
```js
const overrideContents = await fs.readFile(gradlePropertiesOverwriteFile, 'utf-8'); // no need for toString() as specifying encoding enforces it returns a String
return fs.appendFile(filePath, `${overrideContents}\n`); // no need to await, can just return the Promise since this function is async
```